src/video_core/texture_cache/texture_cache_base.h

```diff
@@ -1,8 +1,12 @@
-// SPDX-FileCopyrightText: Copyright 2019 yuzu Emulator Project
-// SPDX-License-Identifier: GPL-2.0-or-later
+// SPDX-FileCopyrightText: 2021 yuzu emulator team
+// (https://github.com/skyline-emu/)
+// SPDX-License-Identifier: GPL-3.0-or-later Licensed under GPLv3
+// or any later version Refer to the license.txt file included.
```

Comment:
Same as above. And the license was changed too

```suggestion
// SPDX-FileCopyrightText: 2021 yuzu Emulator Project
// SPDX-FileCopyrightText: 2021 Skyline Team and Contributors <https://github.com/skyline-emu>
// SPDX-License-Identifier: GPL-3.0-or-later
```